### ISSUE TITLE
[BUG] nothing works anymore

### ISSUE THREAD

**Issue by Marrcoooo** [pizzaboxer/bloxstrap#801]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

Hi, i need some help with bloxtrap
for me bloxtrap isn't working anymore (i mean it still launchs roblox but fastflags (fps unlocker..) and mods  (old backround, font..) aren't working anymore.

**Comment by pizzaboxer** [comment]
Launch Roblox with Bloxstrap manually, explained by https://github.com/pizzaboxer/bloxstrap/wiki/Switching-between-Roblox-and-Bloxstrap

**Comment by Marrcoooo** [comment]
Hello
Thanks for your help but I've already tried to launch bloxtrap this way
Bloxstrap launches normally, it gives me server information and discord status but Mods and Fastflags still aren't working

**Comment by pizzaboxer** [comment]
Try and launch Roblox, then get the latest log file from `%localappdata%\Bloxstrap\Logs` and send it here, should be right at the top.

**Comment by Marrcoooo** [comment]
okay [Bloxstrap_20231023T004457Z.log](https://github.com/pizzaboxer/bloxstrap/files/13065074/Bloxstrap_20231023T004457Z.log)

**Comment by pizzaboxer** [comment]
It seems to think Roblox is already running, so it skips the mod check. Check to make sure Roblox isn't running first in Task Manager, then launch Roblox. Though, I'm assuming by this point it should be fixed by now.

**Comment by Marrcoooo** [comment]
Yes, roblox was running in the background for some reasonq but now it's fixed, thanks for your help!
